### PR TITLE
Configure docker

### PR DESCRIPTION
**Pull request by k0va1**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 RUN mkdir -p $APP_HOME
 
 WORKDIR $APP_HOME
 
+COPY Gemfile Gemfile.lock $APP_HOME/
+
 RUN bundle config --global git.allow_insecure true && \
-    bundle config github.https true
+    bundle config github.https true && \
+    bundle install
```
Comment (IvanShamatov) [inline Dockerfile]: I guess we need to change directory at some point to be able to `bundle install`

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+
+COPY . .
```
Comment (IvanShamatov) [inline Dockerfile:17]: why do we need this if we can copy everything in line 11?

**Review by IvanShamatov — COMMENTED** [review]
@k0va1 Can you clarify a couple of questions below? Thanks

File: Dockerfile (hunk 2 above)
Comment (k0va1) [reply Dockerfile:17]: yeah, we could copy everything in line 11, but if something in `app` directory is changed the `bundle install` will be call again. [here](https://medium.freecodecamp.org/dockerfile-copy-vs-add-key-differences-and-best-practices-9570c4592e9e)(COPY section) is the more complete explanation

**Pull request merged**